x-pack/plugins/ml/public/application/datavisualizer/file_based/components/import_view/import_view.js

```diff
@@ -215,7 +215,7 @@ export class ImportView extends Component {
                 // mappings, use this field as the time field.
                 // This relies on the field being populated by
                 // the ingest pipeline on ingest
-                if (mappings[DEFAULT_TIME_FIELD] !== undefined) {
+                if (mappings.properties[DEFAULT_TIME_FIELD] !== undefined) {
```

Comment:
@peteharverson This change seems to fix the problem. I suspect this problem was introduced separate from index pattern changes. Also, I'm uncertain if anything else should be looked at due to this change.

Reply:
This bug was introduced with the changes in elastic/elasticsearch#62158 and #77035

Reply:
Thanks for fixing the setting of the time field in https://github.com/elastic/kibana/pull/77791/commits/dd2caff346695651f1d3f3a3b0aab1977abeb18f. Tested the import, and the time field is set correctly in the index pattern. I think this might have been broken by the changes in #77035.